lib/active_shipping/carriers/ups.rb

```diff
@@ -65,8 +65,11 @@ class UPS < Carrier
       "83" => "UPS Today Dedicated Courier",
       "84" => "UPS Today Intercity",
       "85" => "UPS Today Express",
-      "86" => "UPS Today Express Saver"
-
+      "86" => "UPS Today Express Saver",
+      "92" => "UPS SurePost (USPS) < 1lb",
+      "93" => "UPS SurePost (USPS) > 1lb",
+      "94" => "UPS SurePost (USPS) BPM",
+      "95" => "UPS SurePost (USPS) Media"
```

Comment:
Want to add the trailing `,` so the next diff doesn't have to touch this line? /nitpick